tests/mcp/kubernetes/k8s.yml

```diff
@@ -0,0 +1,276 @@
+components:
```

Comment:
what's the difference between files in this directory and ones in `tests/mcp/deployment/k8s/`

Reply:
This one is used for the unit tests, with all adapters being used for `test_app_1` wheras the ones in `test/mcp/deployment/k8s` are different apps and for testing successful deployments